File: GazeChim.Data/Models/BsffPackagingAcceptation.cs
using GazeChim.Data.Models.Enums;

namespace GazeChim.Data.Models
{
    public class BsffPackagingAcceptation
    {
        public DateTime Date { get; set; }
        public float Weight { get; set; }
        public WasteAcceptationStatus Status { get; set; }
        public string RefusalReason { get; set; }
        public string WasteCode { get; set; }
        public string WasteDescription { get; set; }
        public Signature Signature { get; set; }
    }
}

File: GazeChim/ClientApp/src/app/models/bsff/packaging/bsff-packaging-acceptation.ts
import { WasteAcceptationStatus } from '../enums/waste-acceptation-status';
import { Signature } from '../signature';

export interface BsffPackagingAcceptation {
  date: Date;
  weight: number;
  status: WasteAcceptationStatus;
  refusalReason: string;
  wasteCode: string;
  wasteDescription: string;
  signature: Signature;
}
